tests/operator/operator.go

```diff
@@ -2354,8 +2354,9 @@ spec:
 
 				By("Expecting to see the profile")
 				Eventually(func() error {
-					_, err = libnode.ExecuteCommandInVirtHandlerPod(nodeName, []string{"/usr/bin/cat", expectedSeccompProfilePath})
-					return err
+					stdout, stderr, err := libnode.ExecuteCommandOnNodeThroughVirtHandler(nodeName, []string{"/usr/bin/cat", expectedSeccompProfilePath})
+					return fmt.Errorf("error=%v, stdout=%s, stderr=%s", err, stdout, stderr)
+
```

Comment:
This is problematic. Let's assume that `err == nil`. If so:
Before: the Eventually clause will return nil.
In this PR: the Eventually clause will return a non-nil error.

Reply:
I'd suggest:
```go
stdout, stderr, err := libnode.ExecuteCommandOnNodeThroughVirtHandler(nodeName, []string{"/usr/bin/cat", expectedSeccompProfilePath})
if err != nil {
	return fmt.Errorf("error=%v, stdout=%s, stderr=%s", err, stdout, stderr)
}
```

Reply:
Thanks for finding this bug of mine.